modules/exploits/linux/ssh/ssh_erlangotp_rce.rb

```diff
@@ -0,0 +1,261 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
```

Comment:
This will automatically run the `check` method before the `exploit` method. 
```suggestion
  include Msf::Auxiliary::Report
  prepend Msf::Exploit::Remote::AutoCheck
```

Reply:
Applied